plugin/web/extensions/sidebar/modals/datacatalog/components/content/DatasetDetails.tsx

```diff
@@ -142,6 +162,12 @@ const ButtonWrapper = styled.div`
   gap: 12px;
 `;
 
+const LayerNamesWrapper = styled.div`
+  justify-content: space-between;
+  gap: 12px;
```

Comment:
I don't think this will have effect unless you make it a flexbox `display: flex;`

Reply:
You're right, My bad.